src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -2030,6 +2029,7 @@ namespace AirflowNetwork {
 
         // Check the number of primary air loops
         if (!simulation_control.DuctLoss) {
+            int NumAPL;
```

Comment:
I would think this should be initialized to 0 here since distribution_simulated could be false? and NumAPL is used at line 2224.